client/package.json

```diff
@@ -12,27 +12,28 @@
   },
   "dependencies": {
     "@types/jest": "^26.0.3",
-    "@vue/cli-service": "~4.3.0",
-    "@vue/cli-service-global": "^4.4.6",
     "@vue/cli-plugin-babel": "~4.3.0",
     "@vue/cli-plugin-eslint": "~4.3.0",
     "@vue/cli-plugin-router": "~4.3.0",
     "@vue/cli-plugin-unit-jest": "^4.4.6",
+    "@vue/cli-service": "~4.3.0",
```

Comment:
I'm confused on what happened here. Did you reorganize these? or update the versions?

Reply:
idk, haha